common-custom-user-data-gradle-plugin/src/main/java/com/gradle/CustomBuildScanEnhancements.java

```diff
@@ -64,7 +64,7 @@ private static void captureCiOrLocal(BuildScanExtension buildScan) {
     }
 
     private static void captureCiMetadata(BuildScanExtension buildScan) {
-        if (isJenkins()) {
+        if (isJenkins() || isHudson()) {
             if (envVariablePresent("BUILD_URL")) {
                 buildScan.link("Jenkins build", envVariable("BUILD_URL"));
```

Comment:
Shouldn't it say `Hudson build` in this new case?

Reply:
Yes, that would probably be preferable. I was hesitant to add an entire new if block since the both set the same environment variables that are captured here (Jenkins is a fork of Hudson after all) but that might not always be the case so I can make a separate condition for Hudson.

Reply:
I went with something in the middle. We can still add a separate section if Jenkins/Hudson diverge from each other.